drivers/hmis/app/graphql/mutations/delete_service.rb

```diff
@@ -6,19 +6,9 @@ class DeleteService < BaseMutation
     field :errors, [Types::HmisSchema::ValidationError], null: false
 
     def resolve(id:)
-      errors = []
-      service = Hmis::Hud::Service.find_by(id: id)
-
-      if service.present?
-        service.destroy
-      else
-        errors << InputValidationError.new("No service found with ID '#{id}'", attribute: 'id')
-      end
-
-      {
-        service: service,
-        errors: errors,
-      }
+      # TODO viewable by
```

Comment:
viewable by should lean on the associated enrollment.

Reply:
Updated https://github.com/greenriver/hmis-warehouse/pull/2521/commits/40ca83ea4470cb461ef4c6fc3fedfe6f5b913175